controllers/admin/AdminImportController.php

```diff
@@ -1596,26 +1596,31 @@ protected function productImportAccessories($offset, $limit, &$crossStepsVariabl
 
     protected function productImportOne($info, $default_language_id, $id_lang, $force_ids, $regenerate, $shop_is_feature_active, $shop_ids, $match_ref, &$accessories, $validateOnly = false)
     {
-        if ($force_ids && isset($info['id']) && (int)$info['id']) {
-            $product = new Product((int)$info['id']);
-        } elseif ($match_ref && array_key_exists('reference', $info)) {
-            $datas = Db::getInstance()->getRow('
-					SELECT p.`id_product`
-					FROM `'._DB_PREFIX_.'product` p
-					'.Shop::addSqlAssociation('product', 'p').'
-					WHERE p.`reference` = "'.pSQL($info['reference']).'"
-				', false);
-            if (isset($datas['id_product']) && $datas['id_product']) {
-                $product = new Product((int)$datas['id_product']);
+        $id_product = null;
```

Comment:
As specified in the label, we should unset the id as soon as the "force IDs" option is disabled. This information mustn't be used.

```php
if (!$force_ids) {
    unset($info['id']);
}
```